recipes/llvm-core/all/conanfile.py

```diff
@@ -65,6 +65,9 @@ class LLVMCoreConan(ConanFile):
         'with_zlib': True,
         'with_xml2': True
     }
+    build_requires = [
+        'cmake/3.20.3'
+    ]
```

Comment:
Please, add a comment close to this build-requirement to tell why it is needed. Our future us will be grateful to our today-us 😄

Reply:
[Done](https://github.com/conan-io/conan-center-index/pull/5720/commits/3ac945755c6900b1584dd82d2c5da4df45cc4aae)!